### PR TITLE
Remove direct references to classes - Fixes #8

### LINKED ISSUE #8: Remove direct reference to classes in mixins

### PR DESCRIPTION
**Pull request by kaelig**
This module has probably never been used in 1.x.x because its documentation was very misleading, and somebody would have noticed if they had used it.

I've fixed the demos and the existing examples in the documentation, and added a few examples to make it clear what's possible to do with this module.

I think it's ready for the tonal treatment that @kavanagh was talking about this morning.

### CODE REVIEW

**Comment by kaelig** [comment]
Looks like @kavanagh is a busy boy, @AlbertoElias can you have a look?

File: README.md [hunk 1]
```diff
@@ -11,45 +11,110 @@ Add the `o-promobox` class to the element you wish to apply the styling.  Typica
...
 ```
 
 Note: it is up to the product developer to decide how to 'flow' other content around the promobox.  They might choose to `float: left;` the promobox or `display: inline` it - or the product developer may decide to display it full width.
 
-## Themeable
-
-Either extend the base classes to create a custom theme or use the [standard theme](http://build.origami.ft.com/files/o-promobox@0.0.7/demos/standard.html).
+## Themes
 
+### [Standard theme](http://build.origami.ft.com/files/o-promobox@1/demos/standard.html)
```
Comment (AlbertoElias) [inline README.md]: why not remove the @1 so it always links to the latest version?

**Comment by AlbertoElias** [comment]
Just a comment, rest lgtm :)

**Comment by matthew-andrews** [comment]
can fill in the background of the history of this module and the reason for:
> This module has probably never been used in 1.x.x because its documentation was very misleading, and somebody would have noticed if they had used it.

I initially hacked it together to try to use origami components for some of the web app's ft weekend stuff — quotes were one of the new components.  After a couple of days of trying I gave up — I think the web app is much more origami compatible than it was back then.  But that should explain why this is how it is and verify that nothing is probably using it.

File: README.md (hunk 1 above)
Comment (kaelig) [inline README.md]: Good point, thanks!

**Comment by kaelig** [comment]
Thanks @matthew-andrews for the background. Not blaming anybody, btw.

Should we delete this module or should we keep it?

**Comment by matthew-andrews** [comment]
would love to be able to use it in next.

**Comment by matthew-andrews** [comment]
[assuming it's in a better state than i left it!]

**Comment by kaelig** [comment]
> [assuming it's in a better state than i left it!]

I'd like to think so but only its usage would prove it. Let me know when you use it and we can decide if we should keep it or kill it if it's not good.

**Comment by kaelig** [comment]
See updated demos: http://registry.origami.ft.com/components/o-promobox@1.2.0

**Pull request merged**